assembly/util/mockVM.ts

```diff
@@ -155,6 +155,30 @@ export namespace MockVM {
     System.putObject(METADATA_SPACE, 'transaction', transaction, protocol.transaction.encode);
   }
 
+  /**
+    * Set transaction that will be used when calling System.getTransaction() and System.getTransactionField(...)
```

Comment:
We should probably update this comment to reflect `getOperation()` looks like it was copied from `getTransaction()` and not updated.

Reply:
updated accordingly